### ISSUE TITLE
Only works with global installation

### ISSUE THREAD

**Issue by mikeerickson** [m-hall/eslint-fixer#9]
Is there someway to define which `eslint` command shall be used.   I prefer to execute using the local `node_modules` version.   As it stands, I have to install the defined rules globally for it to work correctly

```
- $ npm i -g babel-eslint
- $ npm i -g eslint-plugin-react
```

**Comment by mikeerickson** [comment]
Just forked this repo and will be adding ability to use local installation instead of global

```
./node_modules/.bin/eslint
```

instead of requiring global installation

```
eslint ...
```

**Comment by mikeerickson** [comment]
@m-hall It seems that you have done some work here to support using local install based on this comment.   How does this plugin determine if using local

```
"support locally installed eslint by setting cwd"
```

**Comment by m-hall** [comment]
@mikeerickson I believe that was someone else who created a pull request. I did test it at the time and it seemed to work.
I had intended to make options so users could specify what source they wanted to use (global, local, custom location) but I never got around to it.

**Comment by mikeerickson** [comment]
Do you happen to know how to get it working (did you merge the PR)?  I have forked and was gonna add it, then I saw that it was "sort of" added.  

My plan was to add as a settings item as you stated above

**Comment by mikeerickson** [comment]
This will be fixed in the next (version >=1.3) release 

![image](https://cloud.githubusercontent.com/assets/183153/22221234/ea89b37a-e166-11e6-9a6a-841fac68aa98.png)
